packages/spanner/system-test/spanner.js

```diff
@@ -1105,7 +1088,9 @@ var spanner = new Spanner(env);
           });
 
           if (attempts < 2) {
-            runOtherTransaction(transaction);
+            runOtherTransaction(function() {
```

Comment:
Can you also add another test, exactly like this, except that instead of sql query you are doing read by specifying keys?